04-onion-routing.md

```diff
@@ -73,42 +72,38 @@ A node:
 
 There are a number of conventions adhered to throughout this document:
 
- - Length: the maximum route length is limited to 20 hops.
  - HMAC: the integrity verification of the packet is based on Keyed-Hash
- Message Authentication Code, as defined by the
- [FIPS 198 Standard](http://csrc.nist.gov/publications/fips/fips198-1/FIPS-198-1_final.pdf)/[RFC 2104](https://tools.ietf.org/html/rfc2104),
- and using a `SHA256` hashing algorithm.
- - Elliptic curve: for all computations involving elliptic curves, the
-   Bitcoin curve is used, as specified in [`secp256k1`](http://www.secg.org/sec2-v2.pdf).
- - Pseudo-random stream: [`ChaCha20`](https://tools.ietf.org/html/rfc7539) is
- used to generate a pseudo-random byte stream. For its generation, a fixed
- null-nonce (`0x0000000000000000`) is used, along with a key derived from a
- shared secret and with a `0x00`-byte stream of the desired output size as the
- message.
-
+   Message Authentication Code, as defined by the [FIPS 198
+   Standard][fips198]/[RFC 2104][RFC2104], and using a `SHA256` hashing
+   algorithm.
+ - Elliptic curve: for all computations involving elliptic curves, the Bitcoin
+   curve is used, as specified in [`secp256k1`][sec2]
+ - Pseudo-random stream: [`ChaCha20`][rfc7539] is used to generate a
+   pseudo-random byte stream. For its generation, a fixed null-nonce
+   (`0x0000000000000000`) is used, along with a key derived from a shared
+   secret and with a `0x00`-byte stream of the desired output size as the
+   message.
  - The terms _origin node_ and _final node_ refer to the initial packet sender
- and the final packet recipient, respectively.
+   and the final packet recipient, respectively.
  - The terms _hop_ and _node_ are sometimes used interchangeably, but a _hop_
- usually refers to an intermediate node in the route rather than an end node.
-        _origin node_ --> _hop_ --> ... --> _hop_ --> _final node_
+   usually refers to an intermediate node in the route rather than an end node.
```

Comment:
I think that prior example was helpful, it aides in identifying that a route with 3 nodes has 2 hops for example.

Reply:
Re-added.